e2e/app.po.ts

```diff
@@ -1,11 +1,119 @@
-import { browser, element, by } from 'protractor';
+import { by, browser, element, ExpectedConditions } from 'protractor';
+import * as webdriver from 'selenium-webdriver';
+import { Promise as P } from 'es6-promise';
```

Comment:
I appreciate meeting about this; it's totally okay for now, and not as many P's as I thought. 👍